src/components/view/ContactInfo/contactInfoEventsCard.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+import { Card, CardHeader, CardText } from 'material-ui/Card';
+import { List, ListItem } from 'material-ui/List';
+
+import { getImageForEnv } from 'static/images/index'
+
+class ContactInfoEventsCard extends Component {
+
+    formatUidToDate = uid => {
+        const splitUid = uid.split('');
+        return `${splitUid.slice(4,6).join('')}/${splitUid.slice(7,9).join('')}/${splitUid.slice(0,4).join('')}`
+    }
+
+    render() {
+        const { events } = this.props
+
+        const cardProps = {
+            initiallyExpanded: true
+        }
+
+        const cardHeaderProps = {
+            title: 'Events',
+            actAsExpander: true,
+            showExpandableButton: true,
+            titleColor: this.props.titleColor
```

Comment:
Why not pull `titleColor` out of `this.props` where you deconstruct `events` in line 17? #recommended